scalding-core/src/main/scala/com/twitter/scalding/TemplateSource.scala

```diff
@@ -1,134 +0,0 @@
-/*
```

Comment:
have looked too much into TempalteSource but it was removed in cascading3 branch from the original scalding PR